TaxCalculator/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.cpal</groupId>
+	<artifactId>TaxCalculator</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
Can we put this a variable in the parent pom

Reply:
Done